addons/Media_Recorder/src_webpack/state/AddonState.jsm

```diff
@@ -46,6 +46,18 @@ export class AddonState {
         });
     }
 
+    //for some reason there is a bug in some lower Safari versions <14, it cause arrayBuffer() undefined
+    //https://gist.github.com/hanayashiki/8dac237671343e7f0b15de617b0051bd
+    _fixArrayBuffer() {
```

Comment:
@maciej-zawlocki-solwit 
Dodanie tutaj definicji metody to jest konsekwencja użycia jej w tickecie #8222 ??

Reply:
Tak. Tam nie zauważyłem, że _fixArrayBuffer nie był dostępny w tym kontekście, bo jak testowałem 8222 to akurat w ten warunek nie wchodziło. To jest poprawka tego błędu.